arch/arm/core/aarch64/arm_mmu.c

```diff
@@ -0,0 +1,464 @@
+/*
+ * Copyright 2019 Broadcom
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <init.h>
+#include <kernel.h>
+#include <arch/arm/aarch64/cpu.h>
+#include <arch/arm/aarch64/arm_mmu.h>
+#include <linker/linker-defs.h>
+#include <sys/util.h>
+
+/* Set below flag to get debug prints */
+#define MMU_DEBUG_PRINTS	0
+/* To get prints from MMU driver, it has to initialized after console driver */
+#define MMU_DEBUG_PRIORITY	70
+
+#if MMU_DEBUG_PRINTS
+/* To dump page table entries while filling them, set DUMP_PTE macro */
+#define DUMP_PTE		0
+#define MMU_DEBUG(fmt, ...)	printk(fmt, ##__VA_ARGS__)
```

Comment:
Any reason to not use the `LOG_*` macros?

Reply:
`MMU_DEBUG `macro is introduced to debug the MMU code itself. For example, to dump page table entries.
I can probably change few prints to use `LOG_ *`macro if not specific to MMU debug.